### PR TITLE
Start of work on creating a Wizard replacement. [Amends #1051]

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

**Comment by humancompanion-usds** [comment]
@k80bowman - Hoping you can review this sometime this week and get the draft pattern out there. I realize it's not ideal to be talking about a pattern on other pages that is not yet published. Makes me wonder if we shouldn't make drafts visible (it's a function of Jekyll so we'd have to use some other mechanism to mark them).

@caw310 - Once it's out there I'd like feedback from the following which they can leave in issue #1051:

- Natalie
- Sean
- Allison
- Josh Kim
- Peter Russo
- Kristen McConnell

They all asked for this or contributed to it in some way. I'll do my best to incorporate their feedback when I get back and update the Wizard pattern to point folks to this pattern instead. It's a draft so you'll need the URL which should end up being: https://design.va.gov/patterns/forms/sub-task

Thank you both! Greatly appreciated. See you soon.

File: src/_patterns/forms/check-a-service-is-suitable.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: pattern
+title: Check a service is suitable
```
Comment (k80bowman) [inline src/_patterns/forms/check-a-service-is-suitable.md:3]: Not a blocker, but could we re-title this "Check that a service is suitable"?

File: src/_patterns/forms/confirmation.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ The form confirmation page consists of:
 5. Action link  
 6. Helpful content  
 
+Examples of [confirmation pages in the wild can be seen here](https://dsva.slack.com/archives/C0NGDDXME/p1579042705077800)
```
Comment (k80bowman) [inline src/_patterns/forms/confirmation.md:29]: Not a blocker, but we should probably not be linking to Slack on our public documentation site. Could we add links to examples here directly?

**Review by k80bowman — APPROVED** [review]

**Comment by caw310** [comment]
I've reached out them to review and add comments to the original ticket. Thanks!

**Pull request merged**